torch/linalg/__init__.py

```diff
@@ -1341,19 +1400,22 @@
 """)
 
 pinv = _add_docstr(_linalg.linalg_pinv, r"""
-linalg.pinv(input, rcond=1e-15, hermitian=False, *, out=None) -> Tensor
+linalg.pinv(A, rcond=1e-15, hermitian=False, *, out=None) -> Tensor
+
+Computes the pseudoinverse (Moore-Penrose inverse) of a matrix.
 
-Computes the Moore-Penrose pseudoinverse of a matrix. Supports batched inputs.
+The pseudoinverse may be `defined algebraically`_
+but it is more computationally convenient to understand it `through the SVD`_
 
-If :attr:`hermitian`\ `= True`, :attr:`input` should be a complex Hermitian or real symmetric
+Supports inputs of float, double, cfloat and cdouble dtypes.
+Also supports batched inputs, and, if the input is batched, the output is batched with the same dimensions.
+
+If :attr:`hermitian`\ `= True`, :attr:`A` should be a complex Hermitian or real symmetric
```

Comment:
again, let's avoid the word "should" - we can say that A is assumed to be complex Hermitian or real symmetric, but this also needs to describe what that assumption entails and what will occur if the matrix is not actually Hermitian

Reply:
Changed to "is assumed to be". Nothing happens when it's not Hermitian. The next sentence explains this

>In this case, this function will just use the lower-triangular half of the matrix.